Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM rust:1.66
+FROM rust:latest AS builder
```

Comment:
I would never use latest since it means irreproducible builds

Reply:
It does not apply in this case, acurast-core does not generate any binary.